.github/workflows/bpmn.workflow.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Validate BPMN'
+on:
+  push:
+    branches: [ master ]
```

Comment:
Misschien hier paths: camunda/bpmn/ zodat dit alleen runt wanneer er een change is in de bpmns

Reply:
@xavierbloemen paths is toegevoegd. Doe nu een merge hiervan